src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -249,7 +249,8 @@ private static CodeActionKind GetCodeActionKindFromSuggestedActionCategoryName(s
                 CodeActionPriority.Low => LSP.PriorityLevel.Low,
                 CodeActionPriority.Medium => LSP.PriorityLevel.Normal,
                 CodeActionPriority.High => LSP.PriorityLevel.High,
-                _ => null // If Roslyn doesn't assign a priority, let LSP decide the priority
+                CodeActionPriority.None => null, // If Roslyn doesn't assign a priority, let LSP decide the priority
```

Comment:
so, tbh, i'm surprised we have a 'none' priority.  i think it's fine for you to do this.  but my actua preference is that we just outright remove this value from roslyn itself.  i see three usages of this, and i don't think any of them should actually exist (they should just not set any value, or set their value to 'medium').

Reply:
I think LSP will need to add a 4th enum value. Will check them whether they are willing to do this.

In the meantime, is this PR considered blocked until then? I've temporarily assigned `Suppress or Configure issues` to `CodeActionPriority.Low`. Is this OK for the meantime, if we add a tracking bug for the 4th enum value to the code?

Reply:
Update, LSP has added a 4th enum type (thanks Miguel!) so I will wait to merge this PR until that change goes in: https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient/pullrequest/280778